packages/shared/chat/sagas.ts

```diff
@@ -289,9 +289,14 @@ function initChatCommands() {
     }
   })
 
-  addChatCommand('showfps', 'Show FPS counter', (_message) => {
+  addChatCommand('debug', 'Show debug panel', (_message) => {
     fpsConfiguration.visible = !fpsConfiguration.visible
     fpsConfiguration.visible ? getUnityInstance().ShowFPSPanel() : getUnityInstance().HideFPSPanel()
+    const versions = injectVersions({})
+    const kernelVersion = versions['@dcl/kernel'] || 'unknown'
+    const rendererVersion = versions['@dcl/unity-renderer'] || 'unknown'
+
+    getUnityInstance().SetLocalDCLVersion(`\nKernel: ${kernelVersion}\nRenderer: ${rendererVersion}`)
```

Comment:
This is being executed in several places. Are you sure there are no other (current) function doing the same?

Reply:
For what I've seen there is not a function doing this atm, I'll re check and in case create one